src/blocks/remote-data-container/components/item-list/ItemList.tsx

```diff
@@ -157,7 +162,7 @@ export function ItemList( props: ItemListProps ) {
 			onChangeView={ onChangeView }
 			paginationInfo={ {
 				totalItems: totalItems ?? data.length,
-				totalPages: totalPages ?? 1,
+				totalPages: totalPages ?? ( hasNextPage ? page + 1 : page - 1 ) ?? 1,
```

Comment:
This doesn't quite work. The part in parentheses will always return a non-null value, so the `1` is never reached

Reply:
Right. What do you think of using `Math.max`?

```js
totalPages: totalPages ?? ( hasNextPage ? page + 1 : Math.max( 1, page ) ),
```

Reply:
I found another issue with the code you are referring to, where instead of navigating to the last result, it returned to the first page. The following commit (and suggestion in above comment) fixes the issue, so I'm curious about your thoughts! @chriszarate 

[`da6fcd1` (#391)](https://github.com/Automattic/remote-data-blocks/pull/391/commits/da6fcd1a830c094855138390169eaff69c06d228)